engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java

```diff
@@ -442,4 +442,4 @@ int getReturnCode() {
 
   }
 
-}
+}
```

Comment:
EOF new line for engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java